drivers/sensor/nordic/qdec_nrfx/qdec_nrfx.c

```diff
@@ -6,6 +6,7 @@
 
 #include <zephyr/drivers/sensor.h>
 #include <zephyr/pm/device.h>
+#include <zephyr/pm/device_runtime.h>
```

Comment:
`device_runtime.h` is not used in this file :) the PM hook and actions are in `<zephyr/pm/device.h>`, if you call `pm_device_runtime_get()` in this file for example you would need `device_runtime.h`

Reply:
removed.